debian/files

```diff
@@ -0,0 +1,2 @@
+amazon-ecr-credential-helper_0.1.0_amd64.buildinfo - extra
```

Comment:
`debian/files` shouldn't be committed at all. https://www.debian.org/doc/debian-policy/ch-source.html#generated-files-list-debian-files

Reply:
Thanks, I wasn't sure about whether `debian/files` and `debian/.debhelper` should be committed.